### ISSUE TITLE
Charm of shrinking made my default size the smaller version

### ISSUE THREAD

**Issue by Tim-Odermatt** [ochotonida/artifacts#337]
**Describe the bug**
My default player size is now 0.5x
with the Charm of Shrinking eqiuped it is 0.25x of the normal player size.

I managed this somehow by loading debuging my inventory items i have lost to a diverant mod. Corail Tombstones (/tbrecovery...)

now I would like to somehow change my player back to the normal 1x height of ca. 2Blocks.

**Game Environment**
Please provide details about the game environment, either by crash report/logs or by detailing it below. Try to reproduce the issue without irrelevant mods first. Otherwise, include a complete list of mods or the name and version of the modpack you're using.
 - Artifacts: 12.0.5
 - Forge/Fabric Api: Neoforge 21.1.73
 - Corail Tombstone the /tbrecovery command which loads a previos state of you inventory+Curios where i did not have the charm equiped

**Comment by Tim-Odermatt** [comment]
![image](https://github.com/user-attachments/assets/b83de068-76ce-4dd4-81fb-f24c3051593a)
Player Size with Charm of Shrinking(what a smol boy)

![image](https://github.com/user-attachments/assets/bed429d3-477b-4294-b801-fa91619c1d94)
now without it equiped

**Comment by ochotonida** [comment]
The attribute modifier applied by the Charm of Shrinking uses a fixed identifier. Even if it were somehow possible to apply two attribute modifiers with the same id, unequipping the item should still remove both modifiers in that case. I don't know what's happening on Corail Tombstone's side since the mod is closed source, but I very much doubt Artifacts is to blame for this.

**Comment by Corail31** [comment]
@ochotonida I don't think that it's related to my mod : attribute modifiers are not serialized for items in vanilla so, restoring saved data shouldn't be problematic (even more after reloading the savegame).
If it can help, i remind that when i did a potion of "little world" and "giant strength", i was forced to change the "operation" of the modifier to "multiplied base" instead of "multiplied total" to avoid a strange issue when clearing the effects.
@Tim-Odermatt you may put all your inventory in a chest (except the charm that you keep equipped) and use /tbrecovery to save your data and display them here (you can find the file in your worldsave in tombstone/saved_players based on the time of the save).
